### PR TITLE
Jaws of life for atmos

### PR DESCRIPTION
**Pull request by FabianK3**
### Introduction
This PR adds a hydraulic rescue tool (jaws of life) to the atmos locker room. The goal is to provide Atmospheric technicians the means to enter areas that they are needed in without knowing the airlock wiring or having access to insulated gloves. This places them in a better position for their emergency response duties. This also provides the opportunity for cross-department work towards paramedics and maybe security.

The idea raised from this suggestion/thread: [Reinforce Engineering's Division of Labor](https://forums.aurorastation.org/topic/21285-reinforce-engineerings-division-of-labor/)

### What has been changed?
- Added a jaws of life to the table in the atmos locker room.
- Added the functionality to force open powered airlocks that are not broken or bolted. Without this functionality the tool wouldn't serve its intended purpose.
- Added a couple of messages when using the tool for different actions for fluff.

### CODE REVIEW

File: maps/sccv_horizon/sccv_horizon.dmm [hunk 1]
```diff
@@ -30029,6 +30029,7 @@
 /obj/item/storage/bag/inflatable,
 /obj/item/storage/bag/inflatable,
 /obj/item/device/hand_labeler,
+/obj/item/crowbar/robotic/jawsoflife,
```
Comment (Generalcamo) [inline maps/sccv_horizon/sccv_horizon.dmm:30032]: Did you forget to commit this file?

**Review by Generalcamo — CHANGES_REQUESTED** [review]

File: maps/sccv_horizon/sccv_horizon.dmm (hunk 1 above)
Comment (Generalcamo) [reply maps/sccv_horizon/sccv_horizon.dmm:30032]: Misread this, carry on

**Review by Generalcamo — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -1465,10 +1465,21 @@ About the new airlock wires panel:
 			cutting = TRUE
 	else if(istype(tool, /obj/item/crowbar/robotic/jawsoflife))
 		if(bolt_cut_state == BOLTS_FINE)
-			to_chat(user, SPAN_WARNING("You force the bolt cover open!"))
-			playsound(src, 'sound/weapons/smash.ogg', 100, extrarange = SHORT_RANGE_SOUND_EXTRARANGE)
-			bolt_cut_state = BOLTS_EXPOSED
+			user.visible_message(\
+				SPAN_DANGER("[user.name] starts using the [tool] on the airlock's bolt cover!"),\
```
Comment (FluffyGhoster) [inline code/game/machinery/doors/airlock.dm:1469]:
```suggestion
				SPAN_DANGER("[user] starts using the [tool] on the airlock's bolt cover!"),\
```

File: code/game/machinery/doors/airlock.dm [hunk 3]
```diff
@@ -1465,10 +1465,21 @@ About the new airlock wires panel:
 			cutting = TRUE
 	else if(istype(tool, /obj/item/crowbar/robotic/jawsoflife))
 		if(bolt_cut_state == BOLTS_FINE)
-			to_chat(user, SPAN_WARNING("You force the bolt cover open!"))
-			playsound(src, 'sound/weapons/smash.ogg', 100, extrarange = SHORT_RANGE_SOUND_EXTRARANGE)
-			bolt_cut_state = BOLTS_EXPOSED
+			user.visible_message(\
```
Comment (FluffyGhoster) [inline code/game/machinery/doors/airlock.dm:1468]: you don't need to make newlines like this, put the first span_danger in the same line and align the others below, and it should work

File: code/game/machinery/doors/airlock.dm [hunk 4]
```diff
@@ -1465,10 +1465,21 @@ About the new airlock wires panel:
...
+				SPAN_DANGER("[user.name] starts using the [tool] on the airlock's bolt cover!"),\
+				SPAN_WARNING("You start applying pressure on the airlock's bolt cover using the [tool]..."),\
+				SPAN_NOTICE("You hear metal cracking and deforming...")\
+			)
+			if (do_after(user, 1 SECONDS))
+				to_chat(user, SPAN_WARNING("You force the bolt cover open!"))
+				playsound(src, 'sound/weapons/smash.ogg', 100, extrarange = SHORT_RANGE_SOUND_EXTRARANGE)
+				bolt_cut_state = BOLTS_EXPOSED
 		else if(bolt_cut_state != BOLTS_FINE)
+			user.visible_message(\
```
Comment (FluffyGhoster) [inline code/game/machinery/doors/airlock.dm:1478]: same as above

File: code/game/machinery/doors/airlock.dm [hunk 5]
```diff
@@ -1465,10 +1465,21 @@ About the new airlock wires panel:
...
+				SPAN_DANGER("[user.name] starts using the [tool] on the airlock's bolts!"),\
```
Comment (FluffyGhoster) [inline code/game/machinery/doors/airlock.dm:1479]:
```suggestion
				SPAN_DANGER("[user] starts using the [tool] on the airlock's bolts!"),\
```

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**